front50-migrations/src/main/java/com/netflix/spinnaker/front50/migrations/RedBlackToBlueGreenK8sPipelinesMigration.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2023 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.front50.migrations;
+
+import com.netflix.spinnaker.front50.api.model.pipeline.Pipeline;
+import com.netflix.spinnaker.front50.model.pipeline.PipelineDAO;
+import java.time.Clock;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Predicate;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+@Component
+@Slf4j
+public class RedBlackToBlueGreenK8sPipelinesMigration implements Migration {
+
+  private static final LocalDate VALID_UNTIL = LocalDate.of(2023, 6, 1);
```

Comment:
Where is that date coming from? Is it a date of a particular release or something else? It's not wrong, I just want to understand that better.

Reply:
I chose it - I am still determining what date we should use here.  I am still not sure if instead, we need to make it enabled all the time (since we do not know when customers will use the released version that contains this migration)

Reply:
Ok, got it! Makes sense!